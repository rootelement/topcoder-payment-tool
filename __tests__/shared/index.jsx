import Application from 'shared';
import React from 'react';

const { JU } = require.requireActual('topcoder-react-utils');

let mockDevFrontendMode = false;

jest.mock('topcoder-react-utils', () => {
  const TRU = require.requireActual('topcoder-react-utils');
  return {
    ...TRU,
    isomorphy: {
      ...TRU.isomorphy,
      isDevBuild: () => mockDevFrontendMode,
    },
  };
});

test('Matches shallow snapshot', () => {
  JU.shallowSnapshot(<Application />);
});

test('Matches shallow snapshot in dev mode', () => {
  mockDevFrontendMode = true;
  JU.shallowSnapshot(<Application />);
});
