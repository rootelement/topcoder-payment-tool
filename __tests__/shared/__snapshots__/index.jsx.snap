// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<div>
  <Connect(MetaTags)
    description="Yet another webapp based on Topcoder React Starter Kit"
    title="Topcoder React Starter"
  />
  <Routes />
</div>
`;

exports[`Matches shallow snapshot in dev mode 1`] = `
<div>
  <Connect(MetaTags)
    description="Yet another webapp based on Topcoder React Starter Kit"
    title="Topcoder React Starter"
  />
  <Routes />
  <DevTools />
</div>
`;
